students/BrianNisonger/session02/series.py

```diff
@@ -0,0 +1,49 @@
+def fibonacci(n):
+	""" compute the nth fibonacci number"""	
+	return sum_series(n,0,1)
+	
+def lucas(n):
+	""" compute the nth Lucas number """
+	return sum_series(n,2,1)
+
+def sum_series(n,n0=0,n1=1):
+    """
+	compute the nth value of a summation series.
+	
+	:param n0=0: value of zeroth element in the series
+	:param n1=1: value of first element in the series
+	
+	This function should generalize the fibonacci() and the lucas(),
+	so that this function works for any first two numbers for a sum series.
+	Once generalized that way, sum_series(n, 0, 1) should be equivalent to fibonacci(n).
+	And sum_series(n, 2, 1) should be equivalent to lucas(n).
+	"""
+    if n==0:
+	    return n0
+    elif n==1:
+        return n1
+    else:
+        return sum_series(n-2,n0,n1)+sum_series(n-1,n0,n1)
+	
+	
+					
+if __name__ == "__main__":
+    # run some tests
+    assert fibonacci(0) == 0
+    assert fibonacci(1) == 1
+    assert fibonacci(2) == 1
+    assert fibonacci(3) == 2
+    assert fibonacci(4) == 3
+    assert fibonacci(5) == 5
+    assert fibonacci(6) == 8
+    assert fibonacci(7) == 13
+    assert lucas(0) == 2
+    assert lucas(1) == 1
+    assert lucas(4) == 7
+    assert sum_series(5) == fibonacci(5)
+    assert sum_series(5, 2, 1) == lucas(5)
+    print("tests passed")
```

Comment:
Looks great!  Nice assert statements!  Just make sure your indentions are 4 spaces and you're good to go.